tests/api/test_recurring_api.py

```diff
@@ -3,108 +3,70 @@
+            "billingCycles": json.dumps([
+                PayPalBillingCycle(
+                    pricing_scheme=PayPalBillingCyclePricingScheme(
+                        fixed_price= PayPalBillingCycleAmount(value="20", currency_code="USD").__dict__
+                    ).__dict__,
+                    frequency= PayPalBillingCycleFrequency(interval_count= 1, interval_unit="MONTH").__dict__,
+                    tenure_type="REGULAR",
+                    sequence=1,
+                    total_cycles=999
+                ).__dict__]
+            ),
+            "paymentPreferences": json.dumps(
+                PayPalPaymentPreferences(
+                    auto_bill_outstanding=True,
+                    setup_fee=PayPalPaymentPreferencesSetUpFee(value=20, currency_code="USD").__dict__,
+                    setup_fee_failure_action="CONTINUE",
+                    Payment_failure_threshold=3
+                ).__dict__
+            ),
+            "taxes": json.dumps(
+                PayPalTaxes(percentage="10",inclusive=True).__dict__
+            ),
+            "productSchema": json.dumps(PayPalProductSchema(type ="SERVICE", category="SOFTWARE").__dict__)
         }
 
-        # Perform the API call through the SDK function
-        response = self.client.auth.auth_voucher_create(params)
+        # Perfo                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    rm the API call through the SDK function
+        response = self.client.recurring.paypal_subscription(params)
         data = response.body
         # Test response code
         self.assertEqual(data['ret_code'], '000100')
 
-unittest = AuthApiTests()
+unittest = RecurringApiTests()
 unittest.setUpClass()
-# unittest.test_auth_capture()
-# unittest.test_auth_detail_query()
-# unittest.test_auth_freeze()
-# unittest.test_auth_unfreeze()
-unittest.test_auth_voucher_create()
+unittest.test_paypal_subscription()
+# unittest.test_pay()
```

Comment:
Remove the commented lines if we don't plan to use it.

Reply:
sure